src/App.js

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
 import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'; // Import Routes and Route
 import Calculator from './components/Calculator';
 import Quote from './components/Quote';
+import Logo from './components/Logo';
+import Navigation from './components/Navigation';
+import Home from './components/Home';
 
 function App() {
   return (
-    <div className="App">
-      <Calculator />
-      <div className="Quote">
-        <Quote />
+    <Router>
+      <div className="App">
+        <div className="App-header">
+          <Logo />
+          <Navigation />
+        </div>
+        <Routes>
+          <Route path="/home" element={<Home />} />
```

Comment:
- [ ] [OPTIONAL]
Good job on the pages design, however, when a user visits the site, it shows a blank page, unless a menu item is clicked. Following best practices, it will be ideal that the index page is set to `Home`  so that the home page is displayed when the user opens the webapp. we can simply do that by modifying the home route like this:

```javascript
 ...
  <Route path="/" element={<Home />} />
 ...
 ```
 
 or
 
 ```jsx
<Route path="/" element={<Layout />}>
   <Route index element={<Home />} />
    <Route path="/calculator" element={<Calculator />} />
    <Route path="/quote" element={<Quotes />} />
</Route>
```

Reply:
Thanks for the feedback! 👍 We'll set the Home page as the default so users see it when they visit the site. Your input is much appreciated! 😊🌐